mm2src/mm2_db/src/indexed_db/drivers/object_store.rs

```diff
@@ -137,6 +137,61 @@ impl IdbObjectStoreImpl {
         Self::items_from_completed_request(&get_request)
     }
 
+    pub async fn count(&self, index_str: &str, index_value: Json) -> DbTransactionResult<usize> {
+        if self.aborted.load(Ordering::Relaxed) {
+            return MmError::err(DbTransactionError::TransactionAborted);
+        }
+
+        let index = index_str.to_owned();
+        let index_value_js =
+            serialize_to_js(&index_value).map_to_mm(|e| DbTransactionError::ErrorSerializingIndex {
```

Comment:
I see this mapping duplicated a lot. Providing a macro can be better for reusability.

Like:
```rs
macro_rules! handle_wasm_error {
    ($fn: expr, $index: expr) => {
        $fn.map_to_mm(|e| DbTransactionError::ErrorSerializingIndex {
            index: $index,
            description: e.to_string(),
        })?
    };
}

let index = index_str.to_owned();
let index_value_js = handle_wasm_error!(serialize_to_js(&index_value), index.clone());
```